tasks/keylime-config.yml

```
@@ -0,0 +1,89 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Make sure verifier snippet directory exists
+  ansible.builtin.file:
```

```suggestion
  file:
```
I know the Ansible docs say "use FQCN everywhere" but we don't do that in the system roles for Ansible builtins. We have an ansible-lint exception for that too - https://github.com/linux-system-roles/keylime_server/blob/main/.ansible-lint#L16. I have it on good authority that using the short name for Ansible built-in modules will continue to work for the foreseeable future, and we will have automation in place to convert all of the system roles if/when we need to convert to using FQCN for Ansible builtins.

---

OK, I changed the instances to not use FQCN. Let me know if it's good now, please.